docs/source/usage/plugins/countParticles.rst

```diff
@@ -33,7 +33,7 @@ Output
 In the output file ``e_macroParticlesCount.dat``, there are three columns.
```

Comment:
you could remove the comma while you're at it ^^;

Reply:
fixed